A job failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/19228900237/job/54962601124) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/54962601124) [(plain)](https://github.com/rust-lang/rust/commit/e0a5fefa55cbc478af45be0a4438823d0dab8df8/checks/54962601124/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
  PR_CI_JOB: 1
  SKIP_SUBMODULES: src/gcc
  IMAGE: tidy
##[endgroup]
error: PRs changing the `nightly` channel should be sent to the `master` branch!
##[error]Process completed with exit code 1.
##[group]Run echo "disk usage:"
echo "disk usage:"
df -h
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
```

</details>

--- a/tests/codegen-llvm/deduced-param-attrs.rs
+++ b/tests/codegen-llvm/deduced-param-attrs.rs
@@ -99,6 +99,7 @@ pub static A1: fn(BigCell) = shared_borrow;
 // The parameter can be mutated through a raw const borrow.
 //
 // CHECK-LABEL: ; deduced_param_attrs::raw_const_borrow
+// CHECK-NEXT:  ;
 // CHECK-NOT:   readonly
 // CHECK-NEXT:  %a)
 #[inline(never)]

--- a/tests/codegen-llvm/deduced-param-attrs.rs
+++ b/tests/codegen-llvm/deduced-param-attrs.rs
@@ -100,7 +100,7 @@ pub static A1: fn(BigCell) = shared_borrow;
 //
 // CHECK-LABEL: ; deduced_param_attrs::raw_const_borrow
 // CHECK-NOT:   readonly
-// CHECK-NEXT : %a)
+// CHECK-NEXT:  %a)
 #[inline(never)]
 pub fn raw_const_borrow(a: Big) {
     black_box(&raw const a);

--- a/tests/mir-opt/gvn_loop.rs
+++ b/tests/mir-opt/gvn_loop.rs
@@ -16,9 +16,9 @@ fn loop_deref_mut(val: &mut Value) -> Value {
     // CHECK-LABEL: fn loop_deref_mut(
     // CHECK: [[VAL_REF:_.*]] = get::<Value>(
     // CHECK: [[V:_.*]] = copy (((*[[VAL_REF]]) as V0).0: i32);
-    // CEHCK-NOT: copy (*[[VAL_REF]]);
+    // CHECK-NOT: copy (*[[VAL_REF]]);
     // CHECK: [[RET:_*]] = Value::V0(copy [[V]]);
-    // CEHCK-NOT: copy (*[[VAL_REF]]);
+    // CHECK-NOT: copy (*[[VAL_REF]]);
     // CHECK: _0 = move [[RET]]
     let val_alias: &Value = get(val);
     let mut stop = false;

